community/detectors/apache_http_server_cve_2021_41773/src/test/java/com/google/tsunami/plugins/detectors/nacos/auth_bypass_lte140/ApacheHttpServerVulnDetectorTest.java

```diff
@@ -0,0 +1,128 @@
+package com.google.tsunami.plugins.detectors.nacos.auth_bypass_lte140;
```

Comment:
Please fix the path and the package name. This is clearly not for Nacos :)

Reply:
fixed,done.